### ISSUE TITLE
Recent Activity not showing recent changes after 7 history entries

### ISSUE THREAD

**Issue by etiology** [snipe/snipe-it#928]
The Dashboard (Recent Activity) view should show the most recent activity at the top of the table.  Right now the most recent activity is shown at the bottom of the table... which can be hundreds of rows deep.  However the GUI will never show this because the table only renders the first (oldest) 7 rows.
**[The bug is in this area of code](https://github.com/snipe/snipe-it/blob/b85d9724884ed364eb54b4d3823d1bb82bcf2945/app/views/backend/dashboard.blade.php#L43-L75)**

Basically the foreach loop needs to iterate through this in reverse order so that the most recent activity is at the **top** of the table and the oldest activity is at the bottom where it isn't rendered.

**Comment by snipe** [comment]
I'm unable to reproduce this on the demo. Most recent shows up at the top, as expected. 

<img width="759" alt="screen shot 2015-07-17 at 12 55 48 pm" src="https://cloud.githubusercontent.com/assets/197404/8755915/406e23b0-2c83-11e5-9d84-01bd318bddad.png">

**Comment by snipe** [comment]
(There is no bug in that section of the code, BTW. The order is set by the query via `Actionlog::orderBy('created_at','DESC')`, so no ordering has to be done in the PHP code.)

**Comment by etiology** [comment]
Oops sorry.  I thought I check this enough before posting this.
